pkg/porter/examples/install_example_test.go

```diff
@@ -14,6 +14,7 @@ func ExamplePorter_install() {
 
 	// Specify any of the command-line arguments to pass to the install command
 	installOpts := porter.NewInstallOptions()
+	// install a bundle with older cnab bundle schema version. It should succeed
```

Comment:
Thanks for the comment here! It helps me understand that this test is also indirectly checking this test case too. 👍